.github/infrastructure/docker-compose-cassandra.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
```

Comment:
I assume this file was committed by mistake and should be removed?

Reply:
I don't know, i assumed it was necessary for the conformance test.
Otherwise, how could it be run ?

This is also referenced in the [point nb 2 of the documentation regarding conformance tests](https://github.com/dapr/components-contrib/blob/master/tests/conformance/README.md#running-conformance-tests)

I tried locally, and indeed the test does fail if i don't run cassandra prior to the test (same for redis statestore conformance test, and just like any other i assume)

![image](https://user-images.githubusercontent.com/1097193/147333147-ef682e17-d0c8-4d7d-b23d-3ab07b5571c2.png)

If i remove this file, what do you suggest then ?

Reply:
ok, my bad. leave as is.